src/SeleniumLibrary/keywords/selectelement.py

```diff
@@ -18,206 +18,196 @@
+        """Returns label of selected option from selection list ``locator``.
+
+        If there are multiple selected options, label of the first option
+        is returned.
 
-        Select list keywords work on both lists and combo boxes. Key attributes for
-        select lists are `id` and `name`. See `introduction` for details about
-        locating elements.
+        See the `Locating elements` section for details about the locator
+        syntax.
         """
         select = self._get_select_list(locator)
         return select.first_selected_option.text
 
     @keyword
     def get_selected_list_labels(self, locator):
-        """Returns the visible labels of selected elements (as a list) from the select list identified by `locator`.
+        """Returns labels of selected options from selection list ``locator``.
 
-        Fails if there is no selection.
+        Starting from SeleniumLibrary 3.0, returns an empty list if there
+        are no selections. In earlier versions this caused an error.
 
-        Select list keywords work on both lists and combo boxes. Key attributes for
-        select lists are `id` and `name`. See `introduction` for details about
-        locating elements.
+        See the `Locating elements` section for details about the locator
+        syntax.
         """
-        select, options = self._get_select_list_options_selected(locator)
-        if not options:
-            raise ValueError("List '%s' does not have any selected values."
-                             % locator)
-        return self._get_labels_for_options(options)
+        options = self._get_selected_options(locator)
+        return self._get_labels(options)
 
     @keyword
     def get_selected_list_value(self, locator):
-        """Returns the value of the selected element from the select list identified by `locator`.
+        """Returns value of selected option from selection list ``locator``.
 
-        Return value is read from `value` attribute of the selected element.
+        If there are multiple selected options, value of the first option
+        is returned.
 
-        Select list keywords work on both lists and combo boxes. Key attributes for
-        select lists are `id` and `name`. See `introduction` for details about
-        locating elements.
+        See the `Locating elements` section for details about the locator
+        syntax.
         """
         select = self._get_select_list(locator)
         return select.first_selected_option.get_attribute('value')
 
     @keyword
     def get_selected_list_values(self, locator):
-        """Returns the values of selected elements (as a list) from the select list identified by `locator`.
+        """Returns values of selected options from selection list ``locator``.
 
-        Fails if there is no selection.
+        Starting from SeleniumLibrary 3.0, returns an empty list if there
```

Comment:
Should this chapter be last lines of the doc string? So that in future they are easy to spot and remove?